src/core/gltf2importer/gltf2parser.cpp

```diff
@@ -864,14 +868,50 @@ void GLTF2Parser::generateTreeNodeContent()
                     }
                 }
 
+                // Create a Morph Controller if we have morph targets
+                {
+                    if (hasMorphTargets) {
+                        // Initiliaze the Morph Controller with the default weights
+                        const QVector<float> nodeDefaultWeight = node.morphTargetWeights;
+                        const QVector<float> meshDefaultWeights = meshData.morphTargetWeights;
+                        const quint8 morphTargetCount = std::min(meshData.morphTargetCount, quint8(8));
+
+                        if (meshData.morphTargetCount > 8)
+                            qCWarning(kuesa) << "Kuesa only supports up to 8 morph targets per mesh";
```

Comment:
does it make sense to continue if we are not going to animate all the morph targets?

Reply:
I think so, at least you'll be able to have something working and the warning, rather than the warning and nothing